### PR TITLE
[cinder-csi-plugin] Example of Cinder CSI PVC Recovery

### LINKED ISSUE #1141: [cinder-csi-plugin][docs] Disaster recovery instruction is missing
**Is this a BUG REPORT or FEATURE REQUEST?**:

 /kind bug

**What happened**:

I try to figure out a disaster recovery concept for Cinder volumes which were previosly attached as PVC on a burn cluster. Imagine the cluster is destroyed (for whatever reasons), the volumes are still in the OpenStack backend present and I know the name/id

**What you expected to happen**:

After create a new cluster I should be able to recover PVCs:

```
---
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: csi-ssd-test
  namespace: demoapp
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
      storage: 25Gi
  storageClassName: sata
  volumeMode: Filesystem
  volumeName: pvc-36b6d348-7eb0-4be9-a324-5315b22dbc04
```
Verified, the volume is in state available in backend.
After apply and try to mount on a Pod the PVC status is Pending.

I tried to set some annotations, like

```
    pv.kubernetes.io/bind-completed: "yes"
    pv.kubernetes.io/bound-by-controller: "yes"
    volume.beta.kubernetes.io/storage-provisioner: cinder.csi.openstack.org
    volume.kubernetes.io/selected-node: vm-k8s-01-ranchernode-3
```
but this gives only PVS status "Lost" and the error message on the attacher:

I0729 09:53:20.468006       1 csi_handler.go:219] Error processing "csi-ssd-test": failed to attach: persistentvolume "pvc-36b6d348-7eb0-4be9-a324-5315b22dbc04" not found

Of course, the volume should be attached to the node, but this makes no difference

**Anything else we need to know?**:

My expectation would be, the csi-attacher is able to connect the volume from the node on the new cluster without copy the data on the node level. Maybe I have overseen the trick.

**Environment**:
- openstack-cloud-controller-manager(or other related binary) version: 1.18
- OpenStack version: unknown
- Others: Ubuntu 18.04 host system

### PR DESCRIPTION
**Pull request by eumel8**
**What this PR does / why we need it**:

documentation example

**Which issue this PR fixes(if applicable)**:
fixes #1141 

```release-note
disaster recovery example of cinder-csi-plugin PVC recovery
```

### CODE REVIEW

File: docs/cinder-csi-plugin/examples.md [hunk 1]
```diff
@@ -254,3 +255,103 @@ $ openstack volume list
 | 07522a3b-95db-4bfa-847c-ffa179d08c39 | pvc-400b1ca8-8786-435f-a6cc-f684afddbbea | available |    1 |                                                 |
 | bf8f9ae9-87b4-42bb-b74c-ba4645634be6 | pvc-4699fa78-4149-4772-b900-9536891fe200 | available |    1 |                                                 |
 ```
+
+## Disaster recovery of PV and PVC
+
+This example assume your Kubernetes cluster is crashed, but your volumes
+are still available in the OpenStack cloud:
```
Comment (jichenjc) [inline docs/cinder-csi-plugin/examples.md:262]: curious, if the original cluster is down, how can we create the pv/pvc yaml file? we have enough info?

Comment (eumel8) [reply docs/cinder-csi-plugin/examples.md:262]: I think you need the volume IDs from OpenStack and puzzle the yaml files together. A backup of this files with etcd backup, rancher-backup, kube-backup would also help.

Comment (jichenjc) [reply docs/cinder-csi-plugin/examples.md:262]: I think the info you provided above might be helpful in the doc as well.. wdyt?

Comment (eumel8) [reply docs/cinder-csi-plugin/examples.md:262]: good idea, have added some notes

**Comment by jichenjc** [comment]
@eumel8 please help to update your release note, thanks

**Comment by eumel8** [comment]
> 
> 
> @eumel8 please help to update your release note, thanks

@jichenjc could you point me to release note or changelog?

**Comment by jichenjc** [comment]
add 

```release-note
NONE
```

image

at the end of your first comment (use `edit`) then you should fix the problem

**Comment by ramineni** [comment]
@eumel8 Thanks for the PR. Have you tested the same in your env? I saw couple of bugs before volume doesnt get attached to diff node , so just checking

**Comment by eumel8** [comment]
> 
> 
> add
> 
> ```
> NONE
> ```
> 
> image
> 
> at the end of your first comment (use `edit`) then you should fix the problem

done

**Comment by eumel8** [comment]
> 
> 
> @eumel8 Thanks for the PR. Have you tested the same in your env? I saw couple of bugs before volume doesnt get attached to diff node , so just checking

yes, of course, it works for me as described

**Comment by jichenjc** [comment]
/lgtm

I think we might need add more detailed steps (in case someone else use this) overtime but so far it's a good start

**Pull request merged**